### PR TITLE
SC-1258 Add copy to clipboard feature to ActionsDropdownItem

### PR DESCRIPTION
**Pull request by jonathan-weibel-sonarsource**
New features:
- Copy to clipboard from an ActionsDropdownItem;
- Popup positioning TopLeft.

They will be used in SonarCloud within the LineOptionsPopup, which open on source viewers, to copy line permalink / line content.

Deployment is needed for https://github.com/SonarSource/sonarcloud-core/pull/497

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sonar-ui-common",
-  "version": "0.0.48",
+  "version": "0.0.49",
```
Comment (gregaubert) [inline package.json]: You should not update the version here yet, I'll show you how to release once the PR is ready.
On the other hand you should update the changelog already.

**Review by gregaubert — CHANGES_REQUESTED** [review]
Just drop the change in the package.json and update the changelog and we will be good to go for me.

**Review by gregaubert — APPROVED** [review]
Looks good!

File: components/ui/popups.tsx [hunk 2]
```diff
@@ -27,7 +27,8 @@ export enum PopupPlacement {
   BottomLeft = 'bottom-left',
   BottomRight = 'bottom-right',
   LeftTop = 'left-top',
-  RightTop = 'right-top'
+  RightTop = 'right-top',
+  TopLeft = 'top-left'
```
Comment (jeremy-davis-sonarsource) [inline components/ui/popups.tsx:40]: I find this a bit confusing, what's the difference between `left-top` and `top-left`?
I guess renaming is a pain, but a jsdoc comment could help?

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM

File: components/ui/popups.tsx (hunk 2 above)
Comment (jonathan-weibel-sonarsource) [reply components/ui/popups.tsx:40]: Hi! I used the same logic than the other positioning, that is to say:

- **left-top** = the popup is positioned on the left side of the block, vertically aligned on top of the block;
- **top-left** = the popup is positioned above the block, horizontally aligned on the left of the block.

Not intuitive, I admit. I'll update the jsdoc.

**Pull request merged**